### ISSUE TITLE
Cannot see all our repositories with BitBucket (Pagination issue?)

### ISSUE THREAD

**Issue by ghuntley** [gitpod-io/gitpod#8650]
Hi

We are connected with BitBucket and we cannot see all our repositories; if I check the network log I can see that the response contains 100 records. I wonder if we are hitting a pagination limit or similar? We currently have 130 repositories and the ones we want to add to GitPod of course are not showing.

Thanks

_Originally posted by @jmortlock in https://github.com/gitpod-io/gitpod/issues/5095#issuecomment-1061381262_

**Comment by princerachit** [comment]
Hi @jmortlock can you share more information like your org/project/username/screenshot etc so that we can investigate and give you a workaround till we solve this issue?

**Comment by jmortlock** [comment]
Hi thanks for the fast reply; I assume you mean our bitbucket info? if so https://bitbucket.org/team-sealink and our gitpod team link https://gitpod.io/t/kelsian/projects

Let me know if you need more info

**Comment by jldec** [comment]
@jmortlock - Question: does this issue prevent you from creating the project entirely?
(So, no way to get the repo to show in the new project repo list, even with search?)

**Comment by jmortlock** [comment]
Yes this stops us from adding the project. Search does not find the repo

**Comment by stuartwan** [comment]
Hi, is there an update to this issue? 

Our team is paying for a few licenses to use Gitpod and this issue is blocking us to use the Team project feature (i.e. we cannot get new team members to take advantage of the Team projects to start working sooner)

Many thanks

**Comment by jldec** [comment]
Another user has reported this. ([internal](https://app.frontapp.com/open/cnv_iwl9oqy))

**Comment by jmls** [comment]
We found an anomaly with the bitbucket sdk  - if the user in question was an administrator then they wouldn't see all the projects

Made them a "normal" developer and everything appeared. YMMV
